huxley/www/js/components/RegistrationView.js

```diff
@@ -39,10 +39,12 @@ var RegistrationView = React.createClass({
   },
 
   getInitialState: function() {
+    var committees = CommitteeStore.getSpecialCommittees();
```

Comment:
Nit: these don't need to be variables, the calls can just be inlined into the returned object.